modules/utils/keyboard_util.py

```diff
@@ -147,32 +147,24 @@ def get_paused_kb(page: int, items_per_page: int) -> InlineKeyboardMarkup:
     return InlineKeyboardMarkup(keyboard)
 
 def get_vote_kb(published_post: Optional[PublishedPost] = None) -> Optional[InlineKeyboardMarkup]:
-    """Generates the InlineKeyboard for the published post and updates the correct number of reactions
+    """Generates the InlineKeyboard for the published post
 
     Args:
         published_post: published post to which the keyboard is attached
 
     Returns:
         new inline keyboard
     """
-    keyboard = []
-    for row in ROWS:  # for each ROW or the keyboard...
-        new_row = []
-        for reaction_id in row:  # ... add all the reactions for that row
-            n_votes = "0" if published_post is None else published_post.get_votes(vote=reaction_id)
-            new_row.append(InlineKeyboardButton(f"{REACTION[reaction_id]} {n_votes}",
-                                                callback_data=f"meme_vote,{reaction_id}"))
-        keyboard.append(new_row)
-
-    # the last button in the last row will be the report button
-    report_button = InlineKeyboardButton("🚩 Report", callback_data="meme_report_spot")
-    if Config.meme_get('report'):
-        if len(keyboard) > 0:
-            keyboard[-1].append(report_button)
-        else:
-            keyboard.append([report_button])
-
-    return InlineKeyboardMarkup(keyboard) if keyboard else None
+    return InlineKeyboardMarkup([
+        *([[InlineKeyboardButton(
+            "🚩 Report",
+            callback_data="meme_report_spot"
```

Comment:
Manterrei comunque il vecchio sistema che supporta anche le reaction manuali gestite dal bot. So che non lo stiamo usando più, però è comunque una feature in più. In alternativa, avrebbe senso rimuovere del tutto la funzionalità, inclusi tutti le impostazioni che si tira dietro